src/libs/actions/User.js

```diff
@@ -119,11 +128,47 @@ function setSecondaryLogin(login, password) {
     });
 }
 
+/**
+ * Validates a login given an accountID and validation code
+ *
+ * @param {Number} accountID
+ * @param {String} validateCode
+ */
+function validateLogin(accountID, validateCode) {
+    let isLoggedIn = sessionAuthToken;
```

Comment:
`isLoggedIn` implies a boolean, so I would suggest either casting to a boolean, or change the name.